src/customTheme.ts

```diff
@@ -46,7 +46,9 @@ const customTheme = extendTheme({
     secondary: blueColor,
   },
   fonts: {
-    heading: '"Open Sans", sans-serif;',
+    heading: '"Inter", sans-serif;',
+    body: '"Inter", sans-serif;',
+    mono: '"Inter", sans-serif;',
```

Comment:
Going through this again, I think we should avoid changing the mono one. This one is used in code blocks, which look weird with Inter font. An example here: https://octoclairvoyant-webapp-git-load-inter-font-octoclairvoyant-team.vercel.app/?repo=testing-library%2Fdom-testing-library&from=v6.11.0&to=latest

What do you think?

Reply:
```suggestion
```

I agree, the Inter font doesn't look nice on the code blocks! The Inter font does not come in a monospaced variant yet, but there's a popular issue for it on their GitHub issue tracker. So maybe my favorite font will be suitable for code-blocks later on!

The current production site uses the system operating font stack, which on macOS is Menlo Regular, and on Windows Segoe UI. Those fonts are not a monospaced font by the way.

I think the proper fix is:

- Revert using Inter for monospaced font with my suggestion
- Review this PR, and merge if happy
- I'll go search for a nice monospaced font that matches well with the Inter font, and make a PR to load in that font

Reply:
Happy to do that!